sos/utilities.py

```diff
@@ -43,7 +43,7 @@
 use for detecting binary files. A less effective method will be used. It is \
 recommended to install proper python3-magic package with the module.
 """
-    log.warning('\n' + fill(msg, 72, replace_whitespace=False) + '\n')
+    log.warning('\n%s\n', fill(msg, 72, replace_whitespace=False))
```

Comment:
could you put the whole thing in an f-string here? would that work?

Reply:
Yes, that should work too. Modified. Thanks.